pkgs/applications/misc/prusa-slicer/default.nix

```diff
@@ -21,33 +24,52 @@
 , ilmbase
 , libpng
 , mpfr
+, nanosvg
 , nlopt
 , opencascade-occt
 , openvdb
 , pcre
 , qhull
-, tbb
-, wxGTK31
+, tbb_2021_8
+, wxGTK32
 , xorg
 , fetchpatch
 , withSystemd ? lib.meta.availableOn stdenv.hostPlatform systemd, systemd
 }:
 let
-  wxGTK-prusa = wxGTK31.overrideAttrs (old: rec {
+  wxGTK-prusa = wxGTK32.overrideAttrs (old: rec {
     pname = "wxwidgets-prusa3d-patched";
-    version = "3.1.4";
+    version = "3.2.0";
+    configureFlags = old.configureFlags ++ [ "--disable-glcanvasegl" ];
+    preConfigure = old.preConfigure + ''
+    sed -ie 's@^\t\(monodll_msw_secretstore\|monolib_msw_secretstore\|basedll_msw_secretstore\|coredll_dark_mode\|monodll_dark_mode\|monolib_dark_mode\|baselib_msw_secretstore\|corelib_dark_mode\)\.o$@\t\1.o \\@' Makefile.in
```

Comment:
This line looks really unmaintainable. We should move this into a patch

Reply:
I've pulled the patch out.